stdlib/2/SimpleHTTPServer.pyi

```diff
@@ -0,0 +1,16 @@
+# Stubs for SimpleHTTPServer (Python 2)
+
+from typing import Any, AnyStr, IO, Mapping, Optional, Union
+import BaseHTTPServer
+from StringIO import StringIO
+
+class SimpleHTTPRequestHandler(BaseHTTPServer.BaseHTTPRequestHandler):
+    server_version = ...  # type: str
+    def do_GET(self) -> None: ...
+    def do_HEAD(self) -> None: ...
+    def send_head(self) -> Optional[IO[str]]: ...
+    def list_directory(self, path: AnyStr) -> Optional[StringIO]: ...
```

Comment:
`TypeVar(str, unicode)` certainly has its uses where you don't want to mix string types together, so it's really just a naming issue. Or it doesn't need to have been a default part of typing, since it's not actually something special.

I'd rather have `AnyStr = Union[str, unicode]` and `EitherStr = TypeVar(str, unicode)`. Kind of hard to distill "exactly one of str or unicode" into a pithy variable name, unfortunately.